iroha/tests/genesis.json

```diff
@@ -0,0 +1,55 @@
+{
+  "txs": [
+    {
+      "isi": [
+      {
+        "Register": {
+          "object": {
+            "Peer": {
+              "id": {
+                "address": "localhost:1337",
+                "public_key": "ed207233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0"
+              },
+              "address": "localhost:1337",
+              "domains": {},
+              "trusted_peers_ids": [],
+              "triggers": [],
+              "parameters": []
+            }
+          },
+          "destination_id": {
+            "DomainName": "global"
+          }
+        }
+      },
+      {
+        "Add": {
+          "object": {
+            "U32": 0
+          },
+          "destination_id": {
+            "DomainName": "global"
+          }
+        }
+      },
+      {
+        "Register": {
+          "object": {
+            "Account": {
+              "id": {
+                "name": "root",
```

Comment:
Please change a name to `alice` (or actually anything else than `root`). Root was a bit of a special case we had, when we had a previous permission system, it is not needed anymore.

Reply:
ok!